tests/test_fierce.py

```diff
@@ -27,6 +27,28 @@ def to_text(self):
         return self.response
 
 
+class TestTraverseExpander(unittest.TestCase):
```

Comment:
We already have a couple `test_traverse_expander_*` tests in `TestFierce`. Could we move these there?

Reply:
Oooh, sure thing. Not sure how I missed that. I'll incorporate the new stuff into there.